### ISSUE TITLE
Using the same tile in different rules (question)

### ISSUE THREAD

**Issue by the-old-one** [Portponky/better-terrain#98]
Hi @Portponky. I wonder if there's a way to use the same tile image with different rules without duplicating it in the sprite sheet?
For example I have a water layer where the same plain square tile needs to be in a few different places to correctly surround the islands.

<img width="196" alt="Screenshot 2024-08-01 162306" src="https://github.com/user-attachments/assets/80da7927-1fe1-4f68-90d2-51b3e0e0851a">

**Comment by Portponky** [comment]
Yes, Godot has a way of doing this. In the TileSet dock, you can right click on a tile and select "Create alternate tile". This makes a different version of the same tile which you can give different properties to. You can make as many alternates of a tile as you like. They use the same image data, so no image duplication happens. You can use the alternate tiles in Better Terrain to give totally different rules to the same tile graphics.

**Comment by the-old-one** [comment]
Now I found it, thanks. I tried it but couldn't find the alternative tiles on the Terrain tab. I've been looking for them next to the original tiles and haven't realised they all are grouped in the top right corner. Which for me is completely obscured by the Inspector panel when on a laptop screen. Screenshots in case someone else will have the same issue.

The Terrain panel on a small screen:
![Screenshot 2024-08-03 093428](https://github.com/user-attachments/assets/03706c18-5ae6-4199-9646-307757c09be0)
The panel extended:
![Screenshot 2024-08-03 093437](https://github.com/user-attachments/assets/8d2cc974-a916-41ce-a8fe-8f49fe43f7f6)
